extensions/amp-consent/0.1/test/test-consent-config.js

```diff
@@ -107,6 +116,7 @@ describes.realWin('ConsentConfig', {amp: 1}, env => {
           'clientConfig': {
             'test': 'ABC',
           },
+          'consentRequired': false,
```

Comment:
do we have another test case that have this overridden to `true`?

Reply:
No, will add test.